packages/block-library/src/image/editor.scss

```diff
@@ -100,7 +74,9 @@ figure.wp-block-image:not(.wp-block) {
 	}
 }
 
-.block-editor-block-list__block[data-type="core/image"] .block-editor-block-toolbar .block-editor-url-input__button-modal {
+.block-editor-block-list__block[data-type="core/image"]
+.block-editor-block-toolbar
+.block-editor-url-input__button-modal {
```

Comment:
Personally, I find this line break confusing because it makes it hard to understand at a glance whether these selectors are one or separate.